### PR TITLE
Update git / workflow / staging-i /

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Update Link Name

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/workflow/staging-i/intent-to-add.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/workflow/staging-i/intent-to-add.md?ref=1347bafb41cc6460888dc405c6a3261536da8e51)

File: git/workflow/staging-i/intent-to-add.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ aspects:
 
 links:
 
-  - '[github.com](https://github.com/jbranchaud/til/blob/master/git/intent-to-add.md){website}'
+  - '[Intent To Add github.com](https://github.com/jbranchaud/til/blob/master/git/intent-to-add.md){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/intent-to-add.md]:
```suggestion
  - '[Intent To Add](https://github.com/jbranchaud/til/blob/master/git/intent-to-add.md){website}'
```

File: git/workflow/staging-i/interactive-staging.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){website}'
+  - '[Interactive Staging Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/interactive-staging.md]:
```suggestion
  - '[Interactive Staging](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){documentation}'
```

File: git/workflow/staging-i/interactively-stage-patches.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){website}'
+  - '[Interactive Staging Official Documentation](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/interactively-stage-patches.md]:
```suggestion
  - '[Interactive Staging](http://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging){documentation}'
```

File: git/workflow/staging-i/interactively-unstage-changes.md [hunk 4]
```diff
@@ -16,8 +16,8 @@ aspects:
 
 links:
 
-  - '[git-scm.com](https://git-scm.com/docs/git-reset){website}'
-  - '[git-scm.com](https://git-scm.com/docs/git-add){website}'
+  - '[Git Reset Official Documentation](https://git-scm.com/docs/git-reset){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/interactively-unstage-changes.md]:
```suggestion
  - '[Git Reset](https://git-scm.com/docs/git-reset){documentation}'
```

File: git/workflow/staging-i/interactively-unstage-changes.md [hunk 5]
```diff
@@ -16,8 +16,8 @@ aspects:
...
+  - '[Git Add Official Documentation](https://git-scm.com/docs/git-add){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/interactively-unstage-changes.md]:
```suggestion
  - '[Git Add](https://git-scm.com/docs/git-add){documentation}'
```

File: git/workflow/staging-i/use-short-status-to-make-output-more-compact.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository){website}'
+  - '[Short Status Official Documentation](http://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository){website}'
```
Comment (kapnobatai137) [inline git/workflow/staging-i/use-short-status-to-make-output-more-compact.md]:
```suggestion
  - '[Short Status](http://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository){documentation}'
```

**Pull request merged**